### PR TITLE
adding r-sbl package

### PR DESCRIPTION
**Pull request by Troelsmou**
Adding r-sbl package

Used the conda skeleton R helper to build the package

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-sbl/GPL-3.txt
Comment (mfansler) [inline recipes/r-sbl/GPL-3.txt:1]: Please remove. We use the copy packaged in `r-base`, staying consistent with CRAN.

File: recipes/r-sbl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - "\"%R%\" -e \"library('sbl')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=sbl
+  license: GPL-3.0-or-later
+  summary: Implements sparse Bayesian learning method for QTL mapping and genome-wide association
+    studies.
+  license_family: GPL3
+  license_file:
+    - 'GPL-3.txt'
```
Comment (mfansler) [inline recipes/r-sbl/meta.yaml:44]: Use `r-base` copy:

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```

File: recipes/r-sbl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('sbl')"           # [not win]
+    - "\"%R%\" -e \"library('sbl')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=sbl
+  license: GPL-3.0-or-later
```
Comment (mfansler) [inline recipes/r-sbl/meta.yaml:39]: DESCRIPTION shows `GPL-3` so

```suggestion
  license: GPL-3.0-only
```

File: recipes/r-sbl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  summary: Implements sparse Bayesian learning method for QTL mapping and genome-wide association
+    studies.
+  license_family: GPL3
+  license_file:
+    - 'GPL-3.txt'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - "Troelsmou"
```
Comment (mfansler) [inline recipes/r-sbl/meta.yaml:49]: Shouldn't need quotes.

```suggestion
    - Troelsmou
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for submitting this! ❤️ 

A few things to touch up, but mostly good.

File: recipes/r-sbl/meta.yaml (hunk 3 above)
Comment (mfansler) [reply recipes/r-sbl/meta.yaml:49]: Also, as a first time contributor, please explicitly comment that you wish to be a maintainer. That is, you have the option to not be a maintainer if you'd like.

**Comment by mfansler** [comment]
Unless a package has compilation, it should be `noarch`. Please remove the skip on Windows and restore the `noarch`. It is fine if it fails the Windows run - only Linux needs to pass.

**Comment by Troelsmou** [comment]
I think this should work now?

**Review by dbast — APPROVED** [review]

**Pull request merged**